docs/adr/0003-event-broker-for-subscription-platform.md

```diff
@@ -0,0 +1,108 @@
+# Event Broker for Subscription Platform
+
+* Deciders: Ben Bangert, Phil Booth, Ryan Kelly, Shane Tomlinson
+* Date: 2019-05-14
+
+## Context and Problem Statement
+
+External OAuth Relying Parties (RPs) that Firefox Account users authenticate
+to need to be kept up to date about whether the user still has an active
+subscription as well as knowing when to delete the user data. For internal
+Mozilla use, Mozilla provides [Firefox Service Notifications][] over SQS
+queues. These events are intended for internal trusted use and not suitable
+for external RPs.
+
+A solution for these concerns is a new system referred to as the FxA Event
+Broker which will store FxA related events and distribute them to relevant
+RPs that the user has accessed via webhooks. The initial version described here applies primarily to external RPs which will only receive subscription status
```

Comment:
Grammar nit: use some commas or a second sentence to explain the webhooks part, otherwise it sounds like the user accessed the RPs via webhooks, which doesn't really make sense :-)

Reply:
Swapped the ordering to better highlight the delivery via webhooks. :grin: